<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncTaskView</class>
 <widget class="QWidget" name="SyncTaskView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QCheckBox" name="chbDateFilter">
     <property name="text">
      <string>Set filter by date</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableView" name="tbvTaskTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QDateEdit" name="deSyncTask"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
